services/tests/test_services_graphql_api.py

```diff
@@ -246,3 +250,144 @@ def test_normal_user_can_query_own_services_gdpr_api_scopes(
     executed = user_gql_client.execute(query)
 
     assert dict(executed["data"]) == expected_data
+
+
+def _set_service_title_and_description(service):
+    service.set_current_language("fi")
+    service.title = "Service title in finnish"
+    service.description = "Service description in finnish"
+    service.save()
+
+    service.set_current_language("en")
+    service.title = "Service title in english"
+    service.description = "Service description in english"
+    service.save()
+
+
+@pytest.mark.parametrize("use_variable", [False, True])
```

Comment:
Not sure what this variable on/off parametrization does in this test. To me it seems that it's testing that one can supply values as variables to a GraphQL query. That doesn't seem too relevant with the subject of this test.  It would simplify this test if this parametrization was removed.

Reply:
I had it as separate test, but earlier many test have been frowned upon, so I merged the tests. can of course separate them again.

Reply:
I meant that you don't have to test at all whether variables can be supplied for a GraphQL query. We can assume that one can. It's something that Graphene (or graphql-core) should handle.

Reply:
It has to be handled in the middleware (I use the `value_from_ast method`) and I didn't at first and noticed only later when tried to use a variable.

I thought it's an error someone could make again later.

Reply:
Ah, I didn't realize that connection. Then it's good.

Reply:
There are two translatable fields in Service: title and description. What if one is translated with a variable and the other with a direct input? Would not need that `use_variable` parametrization.

Reply:
I wanted to leave the other without directive. But I can add an alias as a third.